### PR TITLE
List Dashboard Resources as a Table

### PR DESCRIPTION
**Pull request by dhaneshnm**
Adds container/component to list resources in a Dashboard.

fixes BlueBrain/nexus/issues/833

### CODE REVIEW

**Comment by dhaneshnm** [comment]
Screenshot_2019-11-19 A knowledge graph for data-driven science

File: src/shared/containers/DashboardResultsContainer.tsx [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+  // build header properties
+  const headerProperties: {
+    title: string;
+    dataIndex: string;
+  }[] =
+    data &&
+    data.head.vars
+      .filter(
+        // we don't want to display total or self url in result table
```
Comment (dhaneshnm) [inline src/shared/containers/DashboardResultsContainer.tsx]: This is copied over from studio prototype. I am not sure if this logic makes sense here. ie, Will always have total, self etc in the result?

File: src/shared/components/ResultsTable/ResultsTable.tsx [hunk 2]
```diff
@@ -0,0 +1,113 @@
+import * as React from 'react';
+import { Table } from 'antd';
+import './ResultTable.less';
+import moment from 'moment';
+import Search from 'antd/lib/input/Search';
```
Comment (julienmachon) [inline src/shared/components/ResultsTable/ResultsTable.tsx]: That's a weird import, isn't it?

I would normally do:
```suggestion
import { Input } from 'antd';

const { Search } = Input;
```

File: src/shared/containers/DashboardResultsContainer.tsx (hunk 1 above)
Comment (julienmachon) [reply src/shared/containers/DashboardResultsContainer.tsx]: Yeh, I think we are going to document this. `self` and `total` are normally used for creating links and pagination respectively. We normally don't need to display them in the Data table.

File: src/shared/containers/DashboardResultsContainer.tsx [hunk 3]
```diff
@@ -0,0 +1,104 @@
+import * as React from 'react';
+import { useNexus } from '@bbp/react-nexus';
+import { Spin, Alert } from 'antd';
+import ResulTable from '../components/ResultsTable/ResultsTable';
```
Comment (julienmachon) [inline src/shared/containers/DashboardResultsContainer.tsx]: typo. It should be 
```suggestion
import ResultsTable from '../components/ResultsTable/ResultsTable';
```

File: src/shared/containers/DashboardResultsContainer.tsx [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+}> = props => {
+  const { loading, data, error } = useNexus<any>(
+    nexus =>
+      nexus.View.sparqlQuery(
+        props.orgLabel,
+        props.projectLabel,
+        encodeURIComponent(props.viewId),
+        props.dataQuery
+      ),
+    [props.dataQuery, props.viewId] // only trigger new call if we have a new query and a new view
```
Comment (julienmachon) [inline src/shared/containers/DashboardResultsContainer.tsx]: There are a lot of API calls being made when changing the dashboard tab. You can simply remove those 2 args from the array it the API call will only be triggered once
```suggestion
    []
```

**Pull request merged**